README.md

```diff
@@ -8,7 +8,11 @@ If you encounter an issue related to these samples please [submit a new issue](h
 ## How to run the JavaScript module sample (Windows 10/Ubuntu 14+/Debian/Raspbian)
 ### Prerequisites
 1. Install latest [Git Client](https://git-scm.com/downloads).
-2. Install latest [Node LTS](https://nodejs.org).
+2. Install latest [Node Current](https://nodejs.org) (v8.4.0).
```

Comment:
We need to think about whether Node LTS is something we require user to install to run node js sample locally.

Reply:
I have verified the current is not required, I will change this back to LTS v6.11.2.